CHANGELOG.md

```diff
@@ -4,11 +4,12 @@ Changes are identified by the date of the released firmware including them. If
 you are running System76 Open Firmware, opening the boot menu will show this
 date followed by an underscore and a short git revision.
 
-## 2022-07-05
+## 2022-07-12
 
-- lemp11: Fix power off under load while on battery power
+- oryp9: Release of open firmware with System76 EC
+- darp8: Fixed power off under load while on battery power
```

Comment:
Did we hold shipping darp8 for this fix?

Reply:
Pretty sure we didn't, so this should be a darp8 update